spec/lobbyist/v2/customer_call_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
```

Comment:
These tests fail for me.  

I get a failure that looks like this:

  1) Lobbyist::V2::CustomerCall#destroy should send a correct request
     Failure/Error: Lobbyist::V2::CustomerCall.destroy(1,1)
     WebMock::NetConnectNotAllowedError:
       Real HTTP connections are disabled. Unregistered request: DELETE http://localhost:3000/v2/companies/1/customer_calls/1.json with headers {'Accept'=>'application/json', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Authorization'=>'Token token="jQuchd091cns"', 'User-Agent'=>'Faraday v0.9.2'}
       
       You can stub this request with the following snippet:
       
       stub_request(:delete, "http://localhost:3000/v2/companies/1/customer_calls/1.json").
         with(:headers => {'Accept'=>'application/json', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Authorization'=>'Token token="jQuchd091cns"', 'User-Agent'=>'Faraday v0.9.2'}).
         to_return(:status => 200, :body => "", :headers => {})
       
       registered request stubs:
       
       stub_request(:delete, "http://localhost:3000/v2/companies/1/customer_calls/1.json").
         with(:headers => {'Accept'=>'application/json', 'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Authorization'=>'Token token="jQuchd091cns"', 'User-Agent'=>'Faraday v0.8.7'})
       
Let's update the tests.